modules/cnv-add-nic-to-vm.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-edit-vms.adoc
+// * cnv/cnv_users_guide/cnv-editing-vm-template
+//
+// Establishing conditionals so content can be re-used for editing VMs
+// and VM templates.
+
+ifeval::["{context}" == "cnv-edit-vms"]
+:object: virtual machine
+:object-gui: Virtual Machines
+endif::[]
+
+ifeval::["{context}" == "cnv-editing-vm-template"]
+:object: virtual machine template
+:object-gui: Virtual Machine Templates
+endif::[]
+
+[id="cnv-vm-add-nic_{context}"]
+
+= Adding a network interface to a {object}
+
+.Procedure
+
+. From the *{object-gui}* tab, select the {object} you created.
+. Select the *Network Interfaces* tab.
+. Click *Add Network Interface*.
+. In the *Add Network Interface* window, specify the *Name*, *Model*, *Network*, *Type*,
+and *MAC Address* of the network interface.
+. Click *Add Network Interface* to add the network interface without restarting the virtual
```

Comment:
I'm having issues in opening the web console, so I can't see or try it for myself, but this section seems weird: The user already clicked "Add Network Interface" 2 sections ago, and in the previous section he was still in the "Add Network Interface" window, so it's seems odd he should be clicking "Add Network Interface" again.
Please re-check.

Reply:
Pasting in Tomas' reply as to the diff bnetween the two versions for my own reference so I can make the accorded changes for 4.3:

"on 4.3 it is:
On the nic list there is an "Add Network Interface" button to open the add nic dialog. After you fill in all the details, you need to submit the dialog; that submit button is called "Add""

Reply:
Here is the 4.3 version:

. From the *{object-gui}* tab, select the {object}.
. Select the *Network Interfaces* tab.
. Click *Add Network Interface*.
. In the *Add Network Interface* window, specify the *Name*, *Model*, *Network*, *Type*,
and *MAC Address* of the network interface.
. Click *Add* to add the network interface.
. Restart the virtual machine to enable access.
. Edit the drop-down lists and check boxes to configure the network
interface.
. Click *Save Changes*.
. Click *OK*.

Reply:
Here is my review after trying this walkthrough on a 4.3 cluster web-console (I don't have any comments on the section before "Click Add Network Interfaces", so I only started quoting from the section on):
> . Click _Add Network Interface_.
The button caption is _Create Network Interface_

> . In the _Add Network Interface_ window, specify the _Name_, _Model_, _Network_, _Type_,
> and _MAC Address_ of the network interface.
This one is correct - the name of the window is indeed _Add Network Interface_.

> . Click _Add_ to add the network interface.
> . Restart the virtual machine to enable access.
> . Edit the drop-down lists and check boxes to configure the network
> interface.
What drop-down lists?
I suspect this sentence refers to the drop-down lists in the _Add Network Interface_ window (Model, Network, Type). In that case - this section is both redundant and misplaced.

> . Click _Save Changes_.
> . Click _OK_.
There are no such buttons (neither _Save Changes_ nor _OK_). The additional NIC setting is applied upon exiting the _Add Network Interface_ window (by pressing _Add_ button) and restarting the VM.

Reply:
Thanks @yossisegev, very helpful! :-)

Bob